envisage/tests/test_egg_plugin_manager.py

```diff
@@ -147,13 +168,11 @@ def test_uses_global_working_set_by_default(self):
             # Create fresh working set for this test, to make sure that the
             # plugin manager picks up the *current* value of
             # pkg_resources.working_set.
-            pkg_resources.working_set = pkg_resources.WorkingSet()
+            fresh_working_set = pkg_resources.WorkingSet()
```

Comment:
Only stylistic changes here; I found the way the test was written previously to be a bit unclear.